.pipelines/prchecks/CveSpecFilePRCheck/OpenAIClientClass.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""
+OpenAIClient adapts Azure OpenAI client requests based on the model configuration.
```

Comment:
Also, post check-in, let's migrate to the non DevTest subscription for this AI service